library/src/main/java/com/novoda/downloadmanager/lib/DownloadsRepository.java

```diff
@@ -49,7 +50,7 @@ public DownloadsRepository(SystemFacade systemFacade, ContentResolver contentRes
             List<FileDownloadInfo> downloads = new ArrayList<>();
             FileDownloadInfo.Reader reader = new FileDownloadInfo.Reader(contentResolver, downloadsCursor);
 
-            while (downloadsCursor.moveToNext()) {
+            while (downloadsCursor != null && downloadsCursor.moveToNext()) {
```

Comment:
Similar to below we can move the null check to line 48 and return `Collections.emptyList()` directly at that point